### PR TITLE
Allow for negative score

### PR DESCRIPTION
**Pull request by VanessaHenderson**
Rather than just matching for <1  I’m not only matching on 1 or -1

### CODE REVIEW

File: src/plusplus.coffee [hunk 1]
```diff
@@ -65,10 +65,10 @@ module.exports = (robot) ->
     # if we got a score, then display all the things and fire off events!
     if score?
       message = if reason?
-                  if reasonScore > 1
-                    "#{name} has #{score} points, #{reasonScore} of which are for #{reason}."
-                  else
+                  if reasonScore == 1 or reasonScore == -1
```
Comment (ajacksified) [inline src/plusplus.coffee]: Oh, while we're at it, could this change to`if 1, name has {score} point, for {reason}`? (singular `point`)

**Comment by VanessaHenderson** [comment]
Done!

**Comment by ajacksified** [comment]
:tada:

**Comment by ajacksified** [comment]
And done, `1.1.5`

**Pull request merged**